Front/client/components/MainButton.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const MainButton = (props) => (
```

Comment:
You could have Button as a component and then use it inside Main avoiding redundant MainButton component